mono/profiler/log.h

```diff
@@ -68,6 +68,10 @@
                added an exception object field to TYPE_CLAUSE
                class unload events no longer exist (they were never emitted)
                removed type field from TYPE_SAMPLE_HIT
+			   Events/feature should be explicitly enabled as by default they won't be emited (use legacy option to get closer)
```

Comment:
This should go in the changelog for the profiler version, not the data version, further up the file.